### PR TITLE
Http server is now using ports from 40000 to 41000

### PR DESCRIPTION
**Pull request by jkonecny12**
It's nice to use specific ports to open all ports on server.

### CODE REVIEW

File: scripts/httpd.py [hunk 1]
```diff
@@ -32,8 +32,17 @@
...
 
+server=None
 # Bind to any free port
-server = HTTPServer(('', 0), SimpleHTTPRequestHandler)
+for port in range(40000, 41000):
+    try:
+        server = HTTPServer(('', port), SimpleHTTPRequestHandler)
+    except OSError:
+        continue
+    break
```
Comment (vpodzime) [inline scripts/httpd.py]: I think this would be better readable if the ``break`` was moved into an ``else`` branch of the try-except block.

**Comment by vpodzime** [comment]
Looks good to me otherwise.

File: scripts/httpd.py (hunk 1 above)
Comment (jkonecny12) [inline scripts/httpd.py]: Nice improvement! I'll do an update.

**Comment by jkonecny12** [comment]
Updated, thank you.

**Pull request merged**